src/components/BorrowerProfile/BorrowerCountry.vue

```diff
@@ -48,28 +42,16 @@ export default {
 			mapAspectRatio: 1.3,
 			mapLat: null,
 			mapLong: null,
-			countryNumLoansFundraising: 0,
-			countryAvgAnnualIncome: '',
-			countryIsoCode: '',
-			countryName: '',
-			countryRegionName: '',
 		};
 	},
 	apollo: {
-		query: gql`query borrowerCountry($loanId: Int!) {
+		query: gql`query borrowerCountryCoords($loanId: Int!) {
```

Comment:
We maintain an initial load of the map coordinates so it can be initialized from the created hook in the client and prepare for the animated map when scrolled into view.